client/me/get-apps/desktop-download-card.jsx

```diff
@@ -0,0 +1,208 @@
+			case 'Linux i686':
+			case 'Linux i686 on x86_64':
+				return translate( 'Requires a 64-bit kernel. ' );
+			default:
+				return translate( 'Requires Windows 7+. ' );
+		}
+	}
+
+	getTranslateComponents( platform ) {
+		switch ( platform ) {
+			case 'MacIntel':
+				return {
+					firstAvailableLink: this.getLinkAnchorTag( WINDOWS_LINK ),
+					secondAvailableLink: this.getLinkAnchorTag( LINUX_TAR_LINK ),
+					thirdAvailableLink: this.getLinkAnchorTag( LINUX_DEB_LINK ),
+				};
+			case 'Linux i686':
+			case 'Linux i686 on x86_64':
+				return {
+					firstAvailableLink: this.getLinkAnchorTag( LINUX_DEB_LINK ),
+					secondAvailableLink: this.getLinkAnchorTag( WINDOWS_LINK ),
+					thirdAvailableLink: this.getLinkAnchorTag( MAC_LINK ),
+				};
+			default:
+				return {
+					firstAvailableLink: this.getLinkAnchorTag( MAC_LINK ),
+					secondAvailableLink: this.getLinkAnchorTag( LINUX_TAR_LINK ),
+					thirdAvailableLink: this.getLinkAnchorTag( LINUX_DEB_LINK ),
+				};
+		}
+	}
+
+	getAlsoAvailableText( platform ) {
+		const { translate } = this.props;
+		switch ( platform ) {
+			case 'MacIntel':
+				return translate( 'Also available for: ' +
+					'{{firstAvailableLink}}Windows{{/firstAvailableLink}}, ' +
+					'{{secondAvailableLink}}Linux (.tar.gz){{/secondAvailableLink}}, ' +
+					'{{thirdAvailableLink}}Linux (.deb){{/thirdAvailableLink}}.',
+					{ components: this.getTranslateComponents( platform ) }
+				);
+			case 'Linux i686':
+			case 'Linux i686 on x86_64':
+				return translate( 'Also available for: ' +
+					'{{firstAvailableLink}}Linux (.deb){{/firstAvailableLink}}, ' +
+					'{{secondAvailableLink}}Windows{{/secondAvailableLink}}, ' +
+					'{{thirdAvailableLink}}Mac{{/thirdAvailableLink}}.',
+					{ components: this.getTranslateComponents( platform ) }
+				);
+			default:
+				return translate( 'Also available for: ' +
+					'{{firstAvailableLink}}MacOS{{/firstAvailableLink}}, ' +
+					'{{secondAvailableLink}}Linux (.tar.gz){{/secondAvailableLink}}, ' +
+					'{{thirdAvailableLink}}Linux (.deb){{/thirdAvailableLink}}.',
+					{ components: this.getTranslateComponents( platform ) }
+				);
+		}
+	}
+
+	getButtonClickHandler( platform ) {
+		const {
+			trackWindowsClick,
+			trackMacClick,
+			trackLinuxTarClick,
+			trackLinuxDebClick,
+		} = this.props;
+
+		switch ( platform ) {
+			case 'MacIntel':
+				return trackMacClick;
+			case 'Linux i686':
+				return trackLinuxTarClick;
+			case 'Linux i686 on x86_64':
+				return trackLinuxDebClick;
+			default:
+				return trackWindowsClick;
+		}
+	}
+
+	getButtonLink( platform ) {
+		switch ( platform ) {
+			case 'MacIntel':
+				return MAC_LINK;
+			case 'Linux i686':
+			case 'Linux i686 on x86_64':
+				return LINUX_TAR_LINK;
+			default:
+				return WINDOWS_LINK;
+		}
+	}
+
+	getCardTitle( platform ) {
+		const { translate } = this.props;
+		switch ( platform ) {
+			case 'MacIntel':
+				return translate( 'Desktop App for Mac' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **7** times:
`translate( 'Desktop App' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).